langchain/src/chat_models/llama_cpp.ts

```diff
@@ -128,6 +133,43 @@ export class ChatLlamaCpp extends SimpleChatModel<LlamaCppCallOptions> {
     }
   }
 
+  async *_streamResponseChunks(
+    input: BaseMessage[],
+    // @ts-expect-error - TS6133: 'options' is declared but its value is never read.
+    options: this["ParsedCallOptions"],
+    runManager?: CallbackManagerForLLMRun
+  ): AsyncGenerator<ChatGenerationChunk> {
+    if (input.length !== 1) {
+      throw new Error("Only one human message should be provided.");
```

Comment:
Is there a way we can fix this for multi-turn messages?

Reply:
I’ll explore this, ‘llama-node-cpp’ sets some of this up explicitly with the instantiation of a session. I should be able to compose a prompt annotated for ‘llama2’, although milage may vary if someone is using a none ‘llama2’ model.